diff --git a/CMakeLists.txt b/CMakeLists.txt
index 58c417a..a866fc7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -7,9 +7,7 @@ file(GLOB AVX512_CODE_IMPL "src/*AVX512.cpp")
 add_library(neo-tmedian SHARED main.cpp src/version.rc ${CODE} ${CODE_IMPL})
 set_property(TARGET neo-tmedian PROPERTY CXX_STANDARD 17)
 
-find_package(Git REQUIRED)
-execute_process(COMMAND ${GIT_EXECUTABLE} describe --first-parent --tags --always OUTPUT_VARIABLE GIT_REPO_VERSION)
-string(REGEX REPLACE "(r[0-9]+).*" "\\1" VERSION ${GIT_REPO_VERSION})
+set(VERSION "2")
 
 configure_file (
   "${PROJECT_SOURCE_DIR}/src/version.hpp.in"
