diff --git a/CMakeLists.txt b/CMakeLists.txt
index a9098f1..9c857c8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -7,14 +7,7 @@ project(
     LANGUAGES CXX
 )
 
-include(FetchContent)
-FetchContent_Declare(
-    libsamplerate
-    GIT_REPOSITORY https://github.com/libsndfile/libsamplerate.git
-    GIT_PROGRESS ON
-)
-
-FetchContent_MakeAvailable(libsamplerate)
+find_package(libsamplerate)
 
 add_library(AudioResample SHARED
     ${CMAKE_SOURCE_DIR}/src/plugin.cpp
